src/js/modules/rpc/server.ts

```diff
@@ -45,6 +55,99 @@ export class ProcessBatchServer extends SupervisorServer {
     }
   }
 
+  enable_coprocessors(input: EnableCoprosRequest): Promise<EnableCoprosReply> {
+    const ids = input.coprocessors.map((script) => script.id);
+    this.logger.info(`request enable wasm script: ${ids}`);
+    const responses = input.coprocessors.map((definition) =>
+      this.validateEnableCoprocInput(definition, this.logger)
+    );
+    return Promise.resolve({ responses });
+  }
+
+  validateEnableCoprocInput(
+    handleDef: EnableCoprocessor,
+    logger: Logger
+  ): EnableCoprocessorRequestData {
+    try {
```

Comment:
Which lines specifically are the ones that might throw an error? It seems this `try` covers too many lines.

Reply:
Then please change it so that the try/ catch only surrounds it, not the whole thing.